### PR TITLE
Create ReactiveObjC.framework and ReactiveObjCBridge.framework

### PR DESCRIPTION
**Pull request by mdiep**
Same drill as #3137, but for ReactiveObjC and ReactiveObjCBridge.

Please review! I’ll merge this at the end of the week unless I receive feedback before then. (I can’t let this take another 3 weeks.)

Once merged, I’ll push the new frameworks to their respective repos and remove them from this one _entirely_!

### CODE REVIEW

**Comment by ikesyo** [comment]
`ReactiveObjC/ReactiveCocoa-Bridging-Header.h` could be removed.

File: .travis.yml [hunk 1]
```diff
@@ -32,6 +32,56 @@ matrix:
         - XCODE_SDK=watchsimulator
         - XCODE_ACTION=build
         - XCODE_DESTINATION="platform=watchOS Simulator,name=Apple Watch - 38mm"
+    - xcode_scheme: ReactiveObjC-macOS
+      xcode_workspace: ReactiveObjC.xcworkspace
+      env:
+        - XCODE_SDK=macosx
+        - XCODE_ACTION="build test"
+        - XCODE_DESTINATION="arch=x86_64"
+        - XCODE_PLAYGROUND_TARGET="x86_64-apple-macosx10.10"
```
Comment (ikesyo) [inline .travis.yml]: `XCODE_PLAYGROUND_TARGET` settings and `script/validate-playground.sh` could/should be removed.

**Comment by ikesyo** [comment]
I got the diff when I opend `ReactiveObjC.xcworkspace` and checked the scheme:

```diff
diff --git a/ReactiveObjC.xcodeproj/xcshareddata/xcschemes/ReactiveObjC-watchOS.xcscheme b/ReactiveObjC.xcodeproj/xcshareddata/xcschemes/ReactiveObjC-watchOS.xcscheme
index e396f10..4193c26 100644
--- a/ReactiveObjC.xcodeproj/xcshareddata/xcschemes/ReactiveObjC-watchOS.xcscheme
+++ b/ReactiveObjC.xcodeproj/xcshareddata/xcschemes/ReactiveObjC-watchOS.xcscheme
@@ -51,6 +51,15 @@
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES">
+      <MacroExpansion>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "A9B315531B3940610001CB9C"
+            BuildableName = "ReactiveObjC.framework"
+            BlueprintName = "ReactiveObjC-watchOS"
+            ReferencedContainer = "container:ReactiveObjC.xcodeproj">
+         </BuildableReference>
+      </MacroExpansion>
       <AdditionalOptions>
       </AdditionalOptions>
    </LaunchAction>
```

**Comment by ikesyo** [comment]
`Result.framework` is included in the `Copy Frameworks` phase of `ReactiveObjC-iOSTests` and `ReactiveObjC-tvOSTests`. Those and `Result.framework` reference in `ReactiveObjc/Supporting Files` group.

**Comment by ikesyo** [comment]
We need this for the failing scheduler tests: https://github.com/Quick/Nimble/pull/334

**Comment by ikesyo** [comment]
@mdiep I'm sorry, my thought isn't correct and the commit 317056a isn't needed.

- Originally the failing test is written as `expect(scheduler.currentDate).to(beCloseTo(NSDate()))` and that would use a Objective-C implementation of Nimble, which evaluates `scheduler.currentDate` eagerly.
- Now the test is written as `expect(scheduler.currentDate).to(beCloseTo(Date(), within: 0.0003))` and that uses the Swift implementation which evaluates `scheduler.currentDate` lazily by `@autoclosure`: 

```swift
func expect<T>(_ expression: @autoclosure @escaping () throws -> T?, file: FileString = #file, line: UInt = #line) -> Expectation<T>
```

I think the difference would be the root cause of the failure.

**Comment by andersio** [comment]
The two tests keep failing on the _RACScheduler gives current date_ test. Hmm.

**Pull request merged**